daemons/compute/server/servers/server_default.go

```diff
@@ -308,6 +308,30 @@ Retry:
 	}, nil
 }
 
+func (s *defaultServer) List(ctx context.Context, req *pb.DataMovementListRequest) (*pb.DataMovementListResponse, error) {
+
+	// Only get the DMs that match the workflow and workflow namespace
+	opts := []client.ListOption{
+		client.MatchingLabels{"dws.cray.hpe.com/owner.namespace": req.Namespace},
+		client.MatchingLabels{"dws.cray.hpe.com/owner.name": req.Workflow},
+	}
+
+	list := dmv1alpha1.RsyncNodeDataMovementList{}
+	if err := s.client.List(ctx, &list, opts...); err == nil {
```

Comment:
Typically we code the exception case inside the if; so if you could invert this it would align better with other code
```go
if err := s.client.List(ctx, &list, opts...); err != nil {
  return nil, err
}
// good case
```